src/NumericalAlgorithms/LinearOperators/Divergence.hpp

```diff
@@ -76,6 +76,15 @@ auto divergence(
         inverse_jacobian) noexcept
     -> Variables<db::wrap_tags_in<Tags::div, FluxTags>>;
 
+/// \ingroup NumericalAlgorithmsGroup
+/// \brief Compute the divergence of the vector `input`
+template <size_t Dim, typename DerivativeFrame>
+Scalar<DataVector> divergence(
```

Comment:
[thinking out loud here] Can you reduce code duplication by calling the tensor-divergence in the implementation of the variables-divergence (in `Divergence.tpp`) ? You'd need to tweak interfaces a little so that you can avoid recomputing the logical partial derivs for each tag in the variables... but perhaps this change is worth thinking about?

Reply:
I'm not sure how because the variables version does not operate on `tnsr::I` as I explained in the reply to your other comment. Have I missed something?

Reply:
No you haven't... I hadn't understood how the variables version was is somewhat more complex. Thanks for clarifying in the other comment :)

Reply:
We can probably also make this function / the new compute tag work with higher-rank tensors. Did you only need the `tnsr::I` version so didn't bother generalizing it or is there another issue?

Reply:
I just want to take the divergence of a vector.

Reply:
I should add, my use case is the `u_\alpha \partial_i v^i_g` source terms. Rather than computing Hessians of all the maps analytically, I'm just doing it numerically.